React/Libraries.md

```diff
@@ -0,0 +1,65 @@
+#### React Libraries
+
+##### React Development Tools
+ - [create-react-app](https://github.com/facebook/create-react-app) - Project setup by running one command
+ - [storybook](https://github.com/storybookjs/storybook) - Development environment for UI components
+
+##### React Frameworks
+- [next.js](https://github.com/vercel/next.js) - The React Framework
+
+##### React Routing
+- [react-router](https://github.com/ReactTraining/react-router) - Declarative routing for React
+
+#### Next.js Routing
+- [next-connect](https://github.com/hoangvvo/next-connect) - Minimal router and middleware layer for Next.js
+
+##### React Styling and Component Libraries
+- [styled-components](https://github.com/styled-components/styled-components) - Visual primitives for the component age
+- [emotion](https://github.com/emotion-js/emotion) - CSS-in-JS library designed for high performance style composition
+- [theme-ui](https://github.com/system-ui/theme-ui) - Library for creating themeable user interfaces based on constraint-based design principles
```

Comment:
ChakraUI is the same thing as ThemeUI but it provides more components, so we can remove ThemeUI to avoid confusion.

Reply:
Removed